staging/src/k8s.io/apiserver/pkg/storage/etcd3/logger.go

```diff
@@ -31,21 +31,23 @@ type klogWrapper struct{}
 
 const klogWrapperDepth = 4
 
+var klogV = klog.V(5)
+
 func (klogWrapper) Info(args ...interface{}) {
-	if klog.V(5).Enabled() {
-		klog.InfoDepth(klogWrapperDepth, args...)
+	if klogV.Enabled() {
+		klogV.InfoSDepth(klogWrapperDepth, fmt.Sprint(args...))
```

Comment:
nit: `fmt.Sprint` not needed

Reply:
It is needed, `args` are not key/value pairs and even if they were we wouldn't have a message string.